src/aiodynamo/credentials.py

```diff
@@ -289,6 +293,81 @@ def and_then(thing: Optional[T], mapper: Callable[[T], U]) -> Optional[U]:
     return thing
 
 
+# https://docs.aws.amazon.com/STS/latest/APIReference/API_AssumeRoleWithWebIdentity.html
+@dataclass
+class AssumeRoleWithWebIdentityCredentials(MetadataCredentials):
+    ACTION = "AssumeRoleWithWebIdentity"
+    VERSION = "2011-06-15"
```

Comment:
these should be module-level constants (with a more descriptive name), since they're not supposed to be changed.